api/envoy/extensions/retry/host/omit_canary_hosts/v3/omit_canary_hosts.proto

```diff
@@ -1,10 +1,11 @@
 syntax = "proto3";
 
-package envoy.config.retry.omit_canary_hosts.v3;
+package envoy.extensions.retry.host.omit_canary_hosts.v3;
```

Comment:
won't this break wire compatibility?

Reply:
I merged this one only a short while back and it's never been used by Envoy, so we can do this.